src/Tests/Meditation.TestUtils/BootstrapNativeHelpers.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.IO;
+using System.Runtime.InteropServices;
+using System;
+
+namespace Meditation.TestUtils
+{
+    public static class BootstrapNativeHelpers
+    {
+        public static string GetMeditationNativeModulePath()
+        {
+            const string netSdkIdentifier = "net8.0";
+            string runtimeIdentifier;
+            string moduleExtension;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && RuntimeInformation.OSArchitecture == Architecture.X64)
+            {
+                runtimeIdentifier = "win-x64";
+                moduleExtension = "dll";
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                throw new NotImplementedException();
```

Comment:
[ToFix] We need an issue number here.

Reply:
There is new Linux milestone: https://github.com/DevToolsNET/meditation/milestone/6